src/vs/workbench/services/actions/common/menusExtensionPoint.ts

```diff
@@ -63,6 +63,21 @@ const apiMenus: IAPIMenu[] = [
 		description: localize('menus.touchBar', "The touch bar (macOS only)"),
 		supportsSubmenus: false
 	},
+	{
```

Comment:
Nit, should we mark this section with `// --- Start Positron ---` etc?

Reply:
We can. I will.